Tests/MountebankSwiftTests/ExampleData/Imposter/Imposter+Examples.swift

```diff
@@ -89,6 +112,60 @@ extension Imposter {
             ]
         )
 
+        static let withExtraOptionsHttp = Example(
+            value: Imposter(
+                port: 19190,
+                networkProtocol: .http,
+                extraNetworkOptions: .http(allowCORS: true),
+                stubs: [],
+                recordRequests: false,
+                numberOfRequests: 0,
+                requests: []
+            ),
+            json: [
+                "port": 19190,
+                "protocol": "http",
+                "allowCORS": true,
+                "stubs": [],
+                "numberOfRequests": 0,
+                "requests": [],
+                "recordRequests": false,
+            ]
+        )
+
+        static let withExtraOptionsHttps = Example(
```

Comment:
Can we also add this to the integration tests so we're sure MB accepts this imposter?